.github/workflows/publish.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish Package to npmjs
+on:
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          cache: yarn
+          node-version: '20.x'
+          registry-url: 'https://registry.npmjs.org'
+          scope: '@airframes'
+      - run: yarn install --frozen-lockfile
+      - run: yarn build
+      - run: yarn npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
@kevinelliott for this to work you'll need to add a NPM automation token as either an org or repo secret. I can change the name if you don't want to add it as something other than `NPM_TOKEN`

Reply:
Ok, let me get that setup!